packages/at_client/lib/src/util/encryption_util.dart

```diff
@@ -7,6 +7,8 @@ import 'package:encrypt/encrypt.dart';
 import 'package:crypto/crypto.dart';
 import 'package:at_utils/at_logger.dart';
 
+//#TODO Replace calls to methods in this class with at_chops methods and
+// move this class to test folder in next major release
```

Comment:
Is it safe to mark this class and all of the methods as `@Deprecated` now?

Reply:
There are 4 sub tasks in https://github.com/atsign-foundation/at_client_sdk/issues/1145
this PR is for first sub task. I am currently working on second sub task.
Once all sub tasks are done, we can mark the methods deprecated